index.js

```diff
@@ -139,11 +154,18 @@ function main() {
         if (!string.includes('=')) {
             continue;
         }
-        const [name, new_value] = string.split('=')
-        const old_value = old_env_vars[name]
+        let [name, new_value] = string.split('=')
+        let old_value = old_env_vars[name]
```

Comment:
Hi. The variable `old_value` will not be changed in each loop, why not keep on using `const` ?

Reply:
No real reason. I just like pretty columns.

```js
let [name, new_value] = ...
let old_value = ...
```
looks a bit neater than
```js
let [name, new_value] = ...
const old_value = ...
```

Strictly speaking `name` also is unchanged so this could have been something like
```js
const [name, new_value] = ...
const old_value = ...

let exported_value
if (isPathVariable(name)) {
    exported_value = filterPathValue(new_value)
} else {
    exported_value = new_value
}
```
or maybe even
```js
const exported_value = maybeFilterPathValue(name, new_value)
```
where maybeFilterPathValue() inspects name and does filtering if necessary.

But this code isn't really that huge so there is no point in trying to enforce some “correct” code style.

Reply:
Got it. Thank you.